lib/src/view/broadcast/broadcast_player_results_screen.dart

```diff
@@ -0,0 +1,363 @@
+                                              ),
+                                              overflow: TextOverflow.ellipsis,
+                                            ),
+                                          ),
+                                        ],
+                                      ),
+                                    ),
+                                  ),
+                                if (player.fideId != null)
+                                  SizedBox(
+                                    width: statWidth,
+                                    child: StatCard(
+                                      'FIDE ID',
+                                      value: player.fideId!.toString(),
+                                    ),
+                                  ),
+                              ],
+                            ),
+                          Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            spacing: cardSpacing,
+                            children: [
+                              SizedBox(
+                                width: statWidth,
+                                child: StatCard(
+                                  context.l10n.broadcastScore,
+                                  value:
+                                      '${player.score!.toStringAsFixed((player.score! == player.score!.roundToDouble()) ? 0 : 1)} / ${player.played}',
+                                ),
+                              ),
+                              if (player.performance != null)
+                                SizedBox(
+                                  width: statWidth,
+                                  child: StatCard(
+                                    context.l10n.performance,
+                                    value: player.performance.toString(),
+                                  ),
+                                ),
+                              if (player.ratingDiff != null)
+                                SizedBox(
+                                  width: statWidth,
+                                  child: StatCard(
+                                    context.l10n.broadcastRatingDiff,
+                                    child: ProgressionWidget(
+                                      player.ratingDiff!,
+                                      fontSize: 18.0,
+                                    ),
+                                  ),
+                                ),
+                            ],
+                          ),
+                        ],
+                      ),
+                    );
+                  }
+
+                  final playerResult = playerResults.games[index - 1];
+
+                  return GestureDetector(
+                    onTap: () {
+                      pushPlatformRoute(
+                        context,
+                        builder: (context) => BroadcastGameScreen(
```

Comment:
Since we can now open a screen from another context, this should be refactored:


https://github.com/lichess-org/mobile/blob/9a6a4004237aac6fac86c87555d7ecf9da7d49dc/lib/src/view/broadcast/broadcast_game_screen.dart#L372-L378

Reply:
I don't understand why this would not work with deep link. The two controllers are independent, they just listen to the same route because there is only one for broadcast.

Reply:
If you can load a game by itself you don't want to load the whole round, right? In a tournament where you have 100 board you'd load 99 boards (and listen to their updates) for nothing.

Reply:
I agree with that but I have done it this way because in the API there are no endpoints to get the player information for a specific board. 

Maybe this refactoring can be done in an other branch ?

Reply:
Yes let's do it in another PR.

Reply:
This code is also problematic because you assume you assume the round controller is loaded: `select((round) => round.requireValue.games[gameId]!)`

I'm not sure it is a good practice to mix `select` and `requireValue` in riverpod. So far we know it is loaded because the only way to access the game screen was through the round screen.

Now that you can access the game screen from the player result screen directly, can you be sure the round screen will be loaded? If you can access a game from another round it won't be the case. This is the main reason why it must be refactored.

Reply:
I removed the `requireValue`

Reply:
After talking with Thibault, it's decided that it is ok to load the full round to display a single game. So no refactoring after all.

But we need to ensure the round controller is loaded before the game controller in the game screen. (So we can do a `requireValue`)